tests/test_log_message.py

```
@@ -0,0 +1,78 @@
+from datetime import datetime
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)